contracts/base/Loans.sol

```diff
@@ -362,14 +364,25 @@ contract Loans is LoansInterface, SignerRole {
         });
     }
 
+    function getAWholeLendingToken() internal view returns (uint256) {
+        uint8 decimals = ERC20Detailed(lendingPool.lendingToken()).decimals();
+        return TEN**decimals;
+    }
+
     function isCollateralSentEnough(
         uint256 msgValue,
         uint256 amountToBorrow,
         uint256 collateralRatio
     ) internal view returns (bool) {
+        // Calculate Collateral Sent (in lending tokens) = ether sent / 1 lending token price in ether -wei(s)- * a whole lending token (with decimals).
+        uint256 aWholeLendingToken = getAWholeLendingToken();
         uint256 oneLendingTokenPriceWeis = uint256(priceOracle.getLatestAnswer());
-        uint256 collateralSent = msgValue.div(oneLendingTokenPriceWeis);
-        uint256 collateralNeeded = amountToBorrow.mul(collateralRatio).div(1000);
+        uint256 collateralSent = msgValue.div(oneLendingTokenPriceWeis).mul(
```

Comment:
I would recommend doing the multiplication before doing the division in solidity, as division loses accuracy.
e.g. in solidity:
10.mul(6).div(3) = 20 but
10.div(3).mul(6) = 18

Reply:
Good catch!